.ruby-version

```diff
@@ -0,0 +1 @@
+2.1.9
```

Comment:
Creating this file also codifies some undocumented developer knowledge required to develop on SIMP.